### DISCUSSION TITLE
Any way to make the Apple TV visible in other HomeKit apps?

### DISCUSSION THREAD

**Discussion by wifidownunder** [maxileith/homebridge-appletv-enhanced#86] [Q&A]
I'm assuming that because the Apple TV is advertised as a set-top-box that it's not possible to have it show in other HomeKit apps (Eve Home, Home+, etc). 
It would be great to be able use the advanced conditions available in the non-Apple HomeKit apps together with this great plugin

**Comment by maxileith** [comment]
Hi @wifidownunder ,

I do not use any other HomeKit apps myself so I did not even notice that. I could imagine that this occurs because the plugin is no longer exposing a set-top box but an actual Apple TV. It may be the case that the third party HomeKit apps filter that type since the vanilla Apple TV cannot do anything anyways. I will try to expose the devices as a set-top box and check if the devices then appear in the third-party apps. If that is the case it may be a good idea to make it configurable wether the devices should be exposed as a set-top box or Apple TV.

**Reply by maxileith** [reply]
I have switched the device to a true set top box in my development environment. However, the devices do both show up in third party apps. This is also true for television devices from other Plugins. It seems like television devices are just not compatible with third party apps in general ...
